### DISCUSSION TITLE
Wishlist: handle unmatched input

### DISCUSSION THREAD

**Discussion by baradhili** [digitalfortress-tech/typeahead-standalone#91] [Ideas]
As a user, I might be say entering a project name, if it doesn't match I would like the control to still do something with it - say create a new project - so that I can simply my UI interactions

**Comment by niketpathak** [comment]
Hello @baradhili , at the moment, it is not possible to add custom html as you desire if no matches are found. You can propose different suggestions (that match closely etc, but it is not possible to add custom html) but we may consider this if more people request such a requirement. thanks!
